### DISCUSSION TITLE
Need to set fixed vehicle break.

### DISCUSSION THREAD

**Discussion by odashevsky** [google/or-tools#3292] [Routing (and legacy CP) questions]
Hello,

I know this might have been discussed, but for the life of me I cannot find answer on this forum. I have a VRP where some vehicles have a known, pre-assigned break at certain times. Meaning, if the vehicle's shift starts at time A and ends at B, there may be a fixed break from known X and Y where both X and Y are between A and B. I know how to mark start and end of the shift, but how do I mark the break starting at X and Y? Again, X and Y are known, I do not need the solver to find ones for me.

Thank you.

**Comment by odashevsky** [comment]
Hi. Bump. Anything at all? Thanks.

**Comment by jmarca** [comment]
Did you try looking at the example program?

https://github.com/google/or-tools/blob/master/ortools/constraint_solver/samples/vrp_breaks.py

James

--

James E. Marca
Activimetrics LLC
